okhttp-tls/build.gradle

```diff
@@ -1,9 +1,13 @@
 apply plugin: 'me.champeau.gradle.japicmp'
 
 jar {
-  manifest {
-    attributes('Automatic-Module-Name': 'okhttp3.tls')
-  }
+  // modify these lines for MANIFEST.MF properties or for specific bnd instructions
+  // are okhttp3.tls.internal.* packages used in dependent libraries?
```

Comment:
Lets talk about this. Can OkHttp libraries use each other’s internal packages?

Reply:
No. In OSGi there is no concept that allows "special" modules to access packages other modules can not. Therefore if any other module needs access to the  okhttp3.tls.internal.* packages we need to export them. If we do not export them, OSGi will detect at resolution time that the package okhttp3.tls.internal is imported from module x but module y does not export it.